docs/marketplace/add_to_marketplace/add_item_by_type/add_sidecar.md

```diff
@@ -4,13 +4,16 @@ title: Publish your sidecar to the Marketplace
 sidebar_label: Create a sidecar
 ---
 
-[Sidecar containers](/development_suite/api-console/api-design/microservice-containers.md#sidecar-containers) are secondary utility containers running side by side with the [main container](/development_suite/api-console/api-design/microservice-containers.md#main-container) in the same host.
+[Sidecar containers](/development_suite/api-console/api-design/microservice-containers.md#sidecar-containers) are secondary utility containers
+running side by side with the [main container](/development_suite/api-console/api-design/microservice-containers.md#main-container) in the same host.
```

Comment:
```suggestion
[Sidecar containers](/development_suite/api-console/api-design/microservice-containers.md#sidecar-containers) are secondary utility containers running side by side with the [main container](/development_suite/api-console/api-design/microservice-containers.md#main-container) in the same host.
```

Reply:
no the line would be too long, it is correct this way.